lib/ansible/plugins/lookup/dnstxt.py

```diff
@@ -26,14 +26,7 @@
 
 - name: iterate over txt entries
   debug: msg="{{item}}"
-  with_dnstxt:
-    - 'test.example.com'
-    - 'other.example.com'
-    - 'last.example.com'
-
-- name: iterate of a comma delimited DNS TXT entry
-  debug: msg="{{item}}"
-  with_dnstxt: "{{lookup('dnstxt', ['test.example.com']).split(',')}}"
+  loop: "{{ q('dnstxt', 'test.example.com', 'other.example.com', 'last.example.com') }}"
```

Comment:
perhaps like:
```yaml
loop: "{{ q('dnstxt', items) }}"
vars:
  items:
  - test.example.com
  - other.example.com
  - last.example.com
```

Would it make sense to add a sibling to `loop` called `items` to avoid the extra `vars` line on many tasks?

Reply:
FYI, we are going to discuss the continued deprecation, and potential UI shortcomings of `loop` during the IRC core team meeting on August 14.